### ISSUE TITLE
\Llineflag

### ISSUE THREAD

**Issue by baem2** [maieul/ledmac#325]
This is a feature request and not that important ...

Would it be possible to have \Llineflag, similar to \Rlineflag, just for the left columns?

**Comment by maieul** [comment]
yes, after releasing of reledmac ;-). I think a will need again many week

**Comment by maieul** [comment]
hi @baem : does this thing important ? because is require to look on many line of code....

**Comment by baem2** [comment]
It would be nice to have; but if it's too much work, it's not that important.
